joern-cli/frontends/c2cpg/src/main/scala/io/joern/c2cpg/astcreation/AstCreatorHelper.scala

```diff
@@ -58,9 +59,25 @@ trait AstCreatorHelper {
     }
   }
 
-  private def fileLines(node: IASTNode): Seq[Int] = {
+  private def fileOffsetTable(node: IASTNode): Array[Int] = {
     val f = fileName(node)
-    global.file2LinesCache.computeIfAbsent(f, _ => IOUtils.readLineLengthsInFile(Paths.get(f)))
+    global.file2OffsetTable.computeIfAbsent(f, _ => genFileOffsetTable(Paths.get(f)))
```

Comment:
Then we could remove `IOUtils.readLineLengthsInFile` as its not needed any longer.